README.md

```diff
@@ -21,13 +17,36 @@ cd swapr-sdk
 Install dependencies
 
 ```sh
-npm install
+yarn install
+```
+
+ Generate GraphQL
+```sh
+yarn codegen
+```
+
+Build
+```sh
+yarn build
 ```
 
+Connect package an app to sdk locally
+```sh
+yarn link
+```
+And on the app repo
+```sh
+yarn link "../swapr-sdk/" #eg. of path to sdk
```

Comment:
```suggestion
yarn link "../swapr-sdk/" #i.e. of path to sdk
```

i.e. means, this is or "isto é"

Reply:
e.g. means "for example", I think it's okay to use it. I can do the proper quotation.

https://www.merriam-webster.com/dictionary/e.g.